rules/falco_rules.yaml

```diff
@@ -2467,7 +2468,8 @@
      quay.io/prometheus-operator/prometheus-operator, k8s.gcr.io/ingress-nginx/kube-webhook-certgen,
      registry.k8s.io/ingress-nginx/kube-webhook-certgen, quay.io/spotahome/redis-operator,
      registry.opensource.zalan.do/acid/postgres-operator, registry.opensource.zalan.do/acid/postgres-operator-ui,
-     rabbitmqoperator/cluster-operator)
+     rabbitmqoperator/cluster-operator, quay.io/kubecost1/opencost-ui, quay.io/kubecost1/kubecost-cost-model,
```

Comment:
can you provide any documentation to attach re the need to have those images as privileged?
`quay.io/kubecost1/opencost-ui, quay.io/kubecost1/kubecost-cost-model`

Reply:
Good question, I checked https://www.opencost.io/docs/ but could not find anything. I removed `opencost-ui` again after checking logs again. The reason I added this here is because I see a lot of these in our logs
```
11:50:12.215643425: Notice Unexpected connection to K8s API Server from container (command=app pid=12774 k8s.ns=monitoring k8s.pod=opencost-6bb8f7dc6c-zz94p container=0a5b27ba8701 image=quay.io/kubecost1/kubecost-cost-model:prod-1.101.2 connection=10.1.x.x:44174->10.0.0.1:443)
```